modules/account-lib/test/unit/coin/algo/transactionBuilder/assetTransferBuilder.ts

```diff
@@ -110,33 +106,29 @@ describe('Algo Asset Transfer Transaction Builder', () => {
       txInfo.genesisHash.should.equal('SGO1GKSzyE7IEPItTxCByw9x8FmnrCDexi9/cOUJOiI=');
     });
 
-    it('should not build a token transaction with an invalid sender address', async function() {
+    it('should not build a token transaction with an invalid sender address', async function () {
       const wrongAddress = 'RIJVLDYWASZZNGOSQNOK7HN6JNFLMMZ3FFBBFG2NNROM5CE744DAJSPZJ';
-      const tx = await txnBuilder.testnet()
+      const tx = await txnBuilder.testnet();
       should.throws(() => tx.sender({ address: wrongAddress }));
     });
 
-    it('should not build a token transaction with an invalid closeRemainderTo address', async function() {
+    it('should not build a token transaction with an invalid closeRemainderTo address', async function () {
       const wrongAddress = 'RIJVLDYWASZZNGOSQNOK7HN6JNFLMMZ3FFBBFG2NNROM5CE744DAJSPZJ';
-      const tx = await txnBuilder.testnet()
+      const tx = await txnBuilder.testnet();
       should.throws(() => tx.closeRemainderTo({ address: wrongAddress }));
     });
 
-    it('should not build a token transaction with an invalid to address', async function() {
+    it('should not build a token transaction with an invalid to address', async function () {
       const wrongAddress = 'RIJVLDYWASZZNGOSQNOK7HN6JNFLMMZ3FFBBFG2NNROM5CE744DAJSPZJ';
-      const tx = await txnBuilder.testnet()
+      const tx = await txnBuilder.testnet();
       should.throws(() => tx.to({ address: wrongAddress }));
     });
 
-    it('should build a valid enable token transaction', async function() {
+    it('should build a valid enable token transaction', async function () {
       const firstRound = 167;
       const lastRound = 1167;
       const fee = 1000;
       const tokenId = 1;
-      const {
-        networks: { testnet },
-      } = AlgoResources;
-      const { genesisHash, genesisID } = testnet;
       const amount = '0';
```

Comment:
Same here

Reply:
This lines weren't used and they be marked by lint as an error